packages/react-composites/src/composites/CallComposite/components/SidePane/usePeoplePane.tsx

```diff
@@ -322,6 +583,59 @@ export const usePeoplePane = (props: {
             onCancel={() => setShowMuteAllPrompt(false)}
           />
         }
+        {
+          /* @conditional-compile-remove(media-access) */
+          <Prompt
```

Comment:
All of this duplication in the template here we should maybe pull this out as its own component

Reply:
Yes, this file is big, has more than 700 lines. Later will need to restructure or make a reusable template prompt component.